user/include/sys/mman.h

```diff
@@ -0,0 +1,12 @@
+#include <stdint.h>
+
+/* Newlib does not provide mmap prototype, so we need to use our own. */
+
+#define MMAP_FLAG_ANONYMOUS 1
```

Comment:
Please use same values as mentioned in [mmap(2)](http://netbsd.gw.com/cgi-bin/man-cgi?mmap), ie. `MAP_ANONYMOUS`, `PROT_WRITE` and so on.